starter-code/css/main.css

```diff
@@ -6,3 +6,220 @@ Green: #00B172
 White: #FFF
 
 */
+
+
+/*****************************************************************
+        RESETS
+*****************************************************************/
+
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+    line-height: 1;
+    font-family: 'Montserrat', sans-serif;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+
+
+/*****************************************************************
+        NAV
+*****************************************************************/
+
+nav {
+   display: flex;
+   background-color: white;
+   height: 14%;
+   width: 100%;
+   position: fixed;
+   top: 0px;
+   box-sizing: border-box;
+   padding: 15px 40px;
+   justify-content: space-between;
+   align-items: center;
+}
+
+nav img {
+    height: 65px;
+}
+
+nav ul {
+    display: flex;
+}
+
+nav ul li {
+    margin: 0 0 0 20px;
+    font-size: 1.35em;
+}
+
+nav ul li a {
+    text-decoration: none;
+    color: black;
+}
+
+
+
+/*****************************************************************
+        HEADER
+*****************************************************************/
+
+header {
+    height: 908px;
+    width: 100%;
+    background-image: url("../images/landing.jpg");
+    background-size: contain;
```

Comment:
On my computer it looks like that your "contain" is not the best choice, when I change it to cover (what I would recommand you) it's better (as the image is now full screen).

But it's the only things I have to say, otherwise it's a really good job !